src/ray/util/logging.cc

```diff
@@ -47,6 +47,14 @@
 
 namespace ray {
 
+std::string GetCallTrace() {
+  std::string return_message = "Cannot get callstack information.";
+#ifdef RAY_USE_GLOG
+  return google::GetStackTraceToString();
```

Comment:
Does it still require us to start glog? How will this look like when it is enabled with spdlog? Are we going to turn both on, but use glog only for collecting stacktrace?

Reply:
> Does it still require us to start glog? How will this look like when it is enabled with spdlog? Are we going to turn both on, but use glog only for collecting stacktrace?

We do not need start glog any more, just call its interface.

Reply:
We still need to setup glog signal handler to log on various signals (e.g., SIGSEGV) right?

Reply:
Nice catch. We may set up a signal handler, but it's not for glog. I'm not sure where signal handler will be used.

Reply:
I think we are currently using `google::InstallFailureSignalHandler();`. We can have further discussion inside the spdlog PR.